thunder/core/prims.py

```diff
@@ -615,6 +616,19 @@ def _check_string_value_meta(s: StringProxy, value: str) -> None:
 )
 
 
+def _check_slice_value_meta(s: AnyProxy, value: slice) -> None:
+    baseutils.check_type(s, AnyProxy)
+    baseutils.check_type(value, slice)
+
+
+check_slice_value = make_prim(
```

Comment:
This is cool and does the trick, but I would prefer to make this into  a general `check_object_equals` (this is because slices can and need to be compared with `==`). This way we can reuse the check when we need to implement something for the next type.

Reply:
Sounds great! Thanks for the detailed feedback!